setup.cfg

```diff
@@ -13,223 +9,25 @@ exclude=.md,
     scripts/oef/launch.py
 max-line-length = 88
 select = B,C,D,E,F,I,W,
-ignore = E203,E501,W503,D202,B014,D400,D401,DAR
+ignore = B014,D202,D400,D401,E501,E203,W503,DAR002,DAR003,DAR101,DAR102,DAR201,DAR202,DAR301,DAR401,DAR402
```

Comment:
Combined `danglint` into `flake8` (and removed standalone darglint). 
Added exceptions for errors for now.